onboarding/scaffolding.py

```diff
@@ -138,6 +138,14 @@ class GraphQlOnboardingScaffolding(graphene.ObjectType):
 
     borough = graphene.String(required=True)
 
+    address_verified = graphene.Boolean(
```

Comment:
Is this really code that's getting added? Was that because it wasn't initially included in the migration to onboarding scaffolding, but it was one of the last vestiges of OnboardingInfo?

Reply:
Blug, yeah, so the short story is that you are correct--our old `session.onboardingStep1` endpoint had an `addressVerified` field that we hadn't migrated.

But the _reason_ we didn't migrate it is because nothing in the front-end actually used it!  But a back-end _test_ used it... I felt weird about removing the relevant logic from the test, but I also felt weird adding the endpoint, so I split the difference and added it, but as a deprecated field.